src/player.cpp

```diff
@@ -5827,7 +5837,7 @@ void player::add_pain_msg(int val, body_part bp) const
     }
 }
 
-static int bound_mod_to_vals(int val, int mod, int max, int min)
```

Comment:
Why are you removing the `static` here? It doesn't use any of the `player` fields or methods.
Do you intend to move the whole function to `Creature`?
If so, it would be a good idea to just replace the body of the function with invocation of the new method - copy+paste is bad design.